<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="programsGroup">
      <property name="title">
       <string>BCI Program Modules</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QListWidget" name="sourceList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="sigprocList">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="applicationList">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="otherGroup">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>10</y>
         <width>161</width>
         <height>160</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>Other</string>
       </property>
       <widget class="QListWidget" name="otherList">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="clearSelectionBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Selection</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="parametersGroup">
        <property name="title">
         <string>Parameters</string>
        </property>
        <widget class="QListWidget" name="paramfilesList">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>241</width>
           <height>91</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="addParmfileBtn">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>20</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
        <widget class="QPushButton" name="removeParmfileBtn">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="saveDirEdit">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>120</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="chooseDirBtn">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>120</y>
           <width>75</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Choose...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="subjectNameEdit">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>150</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="subjectSessionEdit">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>150</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="statusGroup">
        <property name="title">
         <string>Status</string>
        </property>
        <widget class="QTextEdit" name="statusEdit">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>104</width>
           <height>64</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Courier</family>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="launchButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>120</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Launch</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBCI2000_Help"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V, Ctrl+S</string>
   </property>
  </action>
  <action name="actionBCI2000_Help">
   <property name="text">
    <string>BCI2000 Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionTo_Others">
   <property name="text">
    <string>to Others</string>
   </property>
  </action>
  <action name="actionTo_Applications">
   <property name="text">
    <string>to Applications</string>
   </property>
  </action>
  <action name="actionTo_Signal_Processing">
   <property name="text">
    <string>to Signal Processing</string>
   </property>
  </action>
  <action name="actionTo_Signal_Source">
   <property name="text">
    <string>to Signal Source</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="text">
    <string>move Up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="text">
    <string>move Down</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
